_posts/2023-12-21-github-discussions.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Moving from Mailing Lists to GitHub Discussions
+---
+
+Dear MDAnalysis community,
+Our Google groups (mailing lists) have been the main way to ask questions, exchange ideas, and to discuss about development. 
+However, with more modern solutions available we felt a change was needed.
+
+The [MDAnalysis Team] considered different alternatives and after lengthy discussions we settled on moving the Google groups to [GitHub Discussions].
+This will have multiple advantages, including proper code formatting and cross-linking between issues and discussions.
+
+[GitHub Discussions] are already enabled on GitHub and can be used right away: **please start using the Discussions forum, instead of the Google groups**.
+
+[GitHub Discussions] are divided into categories, and we will have the following (and more):
+
+[Q&A]: for asking questions about MDAnalysis usage, _the equivalent of the “mdnalysis-discussion“ mailing list_
+[Development](https://github.com/MDAnalysis/mdanalysis/discussions/categories/developement): for discussions about MDAnalysis development, _the equivalent of the “mdnalysis-develop” mailing list_
```

Comment:
```suggestion
[Development](https://github.com/MDAnalysis/mdanalysis/discussions/categories/developement): for discussions about MDAnalysis development, _the equivalent of the [mdnalysis-devel][] mailing list_
```

Ouch, I just realised that the category on GitHub Discussions has an extra e (should be development, not developEment) -- would it be possible to change this given that it's still early on?

I believe the mailing list was also `-devel`, too